### PR TITLE
Suspend scaling policies support

### PR DESCRIPTION
**Pull request by suganyakrishnan**
Java SDK has been implemented for
- Suspend scaling policies
- Remove suspended scaling policies
- List all suspended scaling policies

### CODE REVIEW

File: src/main/java/com/spotinst/sdkjava/model/SpotinstElastigroupRepo.java [hunk 1]
```diff
@@ -340,4 +347,62 @@
 
         return retVal;
     }
+
+    @Override
+    public RepoGenericResponse<SuspendedScalingPolicy> suspendScalingPolicies(String groupId, String policyName,
```
Comment (anuragsharma-123) [inline src/main/java/com/spotinst/sdkjava/model/SpotinstElastigroupRepo.java:352]: Change 'groupId' parameter to to 'elastigroupId' for consistency.

**Review by anuragsharma-123 — CHANGES_REQUESTED** [review]
Minor changes requested. Please review.

File: src/main/java/com/spotinst/sdkjava/model/SpotinstElastigroupRepo.java (hunk 1 above)
Comment (suganyakrishnan) [reply src/main/java/com/spotinst/sdkjava/model/SpotinstElastigroupRepo.java]: Incorporated

**Review by anuragsharma-123 — APPROVED** [review]
Reviewed the changes. Pushing for Ziv's approval.

**Pull request merged**